README.md

```diff
@@ -159,24 +159,23 @@ To check that it's working you can inspect the replication log to find `using pa
 
 (You can also verify if the cache is working by inspecting the time each layer takes to encode, `encoding, layer:` in the log, where the first two layers, forward and reverse, will take more time than the rest to populate the cache while the remaining 8 should see a considerable time drop.)
 
-### Memory
-
-We try to generate the MTs in parallel to speed up the process but that takes 2 sector sizes per each layer (e.g., a 1 GiB sector may require, in the worst case scenario, up to 20 GiB of memory to hold the MTs alone). To reduce that (at the cost of speed) we have the (experimental) `disk-trees` feature to offload the MTs to disk when we don't use them. For example, to run the `zigzag` example with this feature you'd need to indicate so to `cargo`,
+In the most extreme case, to reduce time at the cost of *a lot* of memory consumption you can turn off the feature that stores MTs on disk (`disk-trees`) to generate them all on RAM and avoid disk I/O (if the HW doesn't have enough RAM to handle the MTs, roughly 20x the sector size, this won't have the desired effect as the OS will start backing them on disk anyway). For example, to run the `zigzag` example with this feature turned off you'd need to indicate so to `cargo`,
```

Comment:
I don't think this is really true, I am not seeing any speed differences between mmemap and disk versions

Reply:
Me neither, but this might also depend on the HW, "the most extreme case" aimed at conveying something like that, could be rephrased.

Reply:
Now that you mention it, actually the mmap-MT should be replaced with the vector-MT (which would be selected only if the disk-trees is explicitly turned off).